.github/ISSUE_TEMPLATE/defect-report.md

```diff
@@ -36,3 +36,23 @@ Please attach the input file(s) that generate this error.  The simpler the input
  - OS: [e.g. iOS]
  - Version: [e.g. 22]
  - Dependencies Installation: [CONDA or PIP]
+
+----------------
+For Change Control Board: Issue Review
+----------------
+This review should occur before any development is performed as a response to this issue.
+- [ ] 1. Is it tagged with a type: defect or task?
```

Comment:
This is a defect report template, should only defect mentioned?

Reply:
It is for NQ1 documentation...I did not change the documentation yet to distingush between the 2 checklists

Reply:
Ok, I could merge it when tests are green.